tethys_cli/gen_commands.py

```diff
@@ -77,23 +77,94 @@ def add_gen_parser(subparsers):
+                                 'KISSINSIGHTS_ACCOUNT_NUMBER, KISSINSIGHTS_SITE_CODE, KISS_METRICS_API_KEY, '
+                                 'MIXPANEL_API_TOKEN, OLARK_SITE_ID, OPTIMIZELY_ACCOUNT_NUMBER, PERFORMABLE_API_KEY, '
+                                 'PIWIK_DOMAIN_PATH, PIWIK_SITE_ID, RATING_MAILRU_COUNTER_ID, SNAPENGAGE_WIDGET_ID, '
+                                 'SPRING_METRICS_TRACKING_ID, USERVOICE_WIDGET_KEY, WOOPRA_DOMAIN, '
+                                 'YANDEX_METRICA_COUNTER_ID. '
+                                 'e.g.: CLICKMAP_TRACKER_ID:123456 CLICKY_SITE_ID:789123')
+    gen_parser.add_argument('--add-backends', dest='add_backends', nargs='+',
+                            help='Add one or more authentication backends to settings.py. Django try these backends in '
+                                 'the same order they are listed. Provide the dot-formatted python path to a custom '
+                                 'backend or one of the following keys: hydroshare, linkedin, google, facebook. '
+                                 'The default backends are django.contrib.auth.backends.ModelBackend and '
+                                 'guardian.backends.ObjectPermissionBackend. '
+                                 'e.g.: project.backend.CustomBackend hydroshare')
+    gen_parser.add_argument('--oauth-options', dest='oauth_options', nargs='+',
+                            help='Add options for oauth providers in the settings.py. '
+                                 'Available options are: SOCIAL_AUTH_GOOGLE_OAUTH2_KEY, '
+                                 'SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET, SOCIAL_AUTH_FACEBOOK_KEY, '
+                                 'SOCIAL_AUTH_FACEBOOK_SECRET, SOCIAL_AUTH_FACEBOOK_SCOPE, '
+                                 'SOCIAL_AUTH_LINKEDIN_OAUTH2_KEY, SOCIAL_AUTH_LINKEDIN_OAUTH2_SECRET, '
+                                 'SOCIAL_AUTH_HYDROSHARE_KEY, SOCIAL_AUTH_HYDROSHARE_SECRET. '
+                                 'e.g.: SOCIAL_AUTH_GOOGLE_OAUTH2_KEY:123456 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET:789123')
+    'channels.layers.InMemoryChannelLayer'
+    gen_parser.add_argument('--channel-layer', dest='channel_layer',
+                            help='Backend to enable communication between apps via websockets. The Default available '
+                                 'value is channels.layers.InMemoryChannelLayer. For production, it is recommended to '
+                                 'install channel_redis and use channels_redis.core.RedisChannelLayer instead. '
+                                 'A custom backend can be added using dot-formatted path.')
+    gen_parser.add_argument('--captcha', dest='captcha', action='store_true',
+                            help='Enable captcha verification. Choices include an image captcha or Google recaptcha. '
+                                 'The Default is False for development environments and True for production '
```

Comment:
The Default is False for development environments and True for production environments. -> The default is True when the --production flag is used and False when it is omitted.